mlflow/store/hdfs_artifact_repo.py

```diff
@@ -159,7 +163,12 @@ def hdfs_system(host, port):
     """
     import pyarrow as pa
 
-    driver = os.getenv('MLFLOW_HDFS_DRIVER') or 'libhdfs'
+    if host and host.startswith("har://"):
+        _logger.info(
+            "Hadoop archive can be open only by libhdfs. Override MLFLOW_HDFS_DRIVER setting.")
```

Comment:
A more accurate message would probably be ".... Ignoring MLFLOW_HDFS_DRIVER setting.

```suggestion
            "Hadoop archive can be open only by libhdfs. Ignoring MLFLOW_HDFS_DRIVER setting.")
```